scripts/uosc.lua

```diff
@@ -369,9 +370,13 @@ function update_human_times()
 		state.time_human = format_time(state.time)
 		if state.duration then
 			local speed = state.speed or 1
-			state.duration_or_remaining_time_human = format_time(
-				options.total_time and state.duration or ((state.time - state.duration) / speed)
-			)
+      if options.total_time or options.destination_time == "total" then
```

Comment:
Since `total_time` is deprecated, code should only work with `destination_time`. Put this:

```lua
if options.total_time and options.destination_time == 'playtime-remaining' then
	options.destination_time = 'total'
end
```

into `scripts/uosc.lua` among other option normalizations (starts at line 108).

And here only check for `options.destination_time`. Start the elseif conditions by checking for the most common `playtime-remaining`, then `total`, and fall to `time-remaining`.

And use single quotes when defining strings.

Reply:
Should I add deprecation warning?

Reply:
>  Start the elseif conditions by checking for the most common playtime-remaining, then total, and fall to time-remaining.

My thinking was that if a user sets invalid value, it will set to a default value of `playtime-remaining`

Reply:
> Should I add deprecation warning?

Sure.

> My thinking was that if a user sets invalid value, it will set to a default value of playtime-remaining

This would be better handled during normalization. So:

```lua
if options.total_time and options.destination_time == 'playtime-remaining' then
	msg.warn('`total_time` is deprecated. Use `destination_time` instead.')
	options.destination_time = 'total'
elseif not itable_index_of({'total', 'playtime-remaining', 'time-remaining'}, options.destination_time) then
	options.destination_time = 'playtime-remaining'
end
```

Reply:
done